src/firefly/js/charts/ui/ChartsContainer.jsx

```diff
@@ -68,7 +68,9 @@ function ensureDefaultChart(tbl_id) {
 function doUpdateViewer(viewerId, tblGroup, chartId, useOnlyChartsInViewer) {
     const currentIds = getViewerItemIds(getMultiViewRoot(), viewerId);
     if (useOnlyChartsInViewer) {
-        dispatchUpdateCustom(viewerId, {activeItemId: currentIds[0]});
+        if (!getActiveViewerItemId(viewerId)) {
+            dispatchUpdateCustom(viewerId, {activeItemId: currentIds[0]});
+        }
```

Comment:
@robyww will you take a look at this?  I think this logic was added for slate.  Will my change cause a regression issue?

Reply:
I can't tell, I think you should build firefly (not irsaviewer) and try the test entry point: firefly/test/tests-viewer.html, then more than one chart. And they place with expanding them.

Reply:
Yes.  They were mentioned under the `More changes here:` section.  You were assigned as reviewer of those PRs as well.

Reply:
Ok, I built it.  https://fireflydev.ipac.caltech.edu/firefly-1099-combined-chart-requrests/firefly
It looks like it worked like before when compared to firefly nightly.